capstone/src/main/java/com/google/sps/Objects/Challenge.java

```diff
@@ -31,27 +33,37 @@ public Badge getBadge() {
     return badge;
   }
 
-  public ArrayList<String> usersCompleted() {
+  public ArrayList<String> getUsersCompleted() {
     return usersCompleted;
   }
 
-  public void addUserCompleted(String userId) {
+  public void addCompletedUser(String userId) {
     this.usersCompleted.add(userId);
   }
 
+  public boolean getIsCompleted(String userId) {
+    if (this.usersCompleted == null) {
+      return false;
+    } else if (this.usersCompleted.contains(userId)) {
+      return true;
+    }
+    return false;
```

Comment:
nit: could simplify by saying `return this.usersCompleted.contains(userId);`

Reply:
good point!